src/nvim/os/shell.c

```diff
@@ -63,6 +63,8 @@ char **shell_build_argv(const char *cmd, const char *extra_args)
 
   rv[i] = NULL;
```

Comment:
I mean, rv[i] is a char pointer. So the correct thing to assign is NULL, not NUL.